portfolio/src/main/webapp/script.js

```diff
@@ -169,6 +169,7 @@ function addMultipleMessagesToDom(messages) {
 
 /**
  * Fetches the message from the JSON server /data and adds it to the DOM.
+ *     Method is called everytime the page is refreshed.
```

Comment:
For this comment I would add a new line in-between as it is extra detail and can be in the next paragraph.

Reply:
In that case if it is a new paragraph it should be prefixed with <p>

Reply:
Understood! Done!